snippets/product-thumbnail.liquid

```diff
@@ -111,9 +110,9 @@
 
   {%- if media.media_type != 'image' -%}
     {%- if media.media_type == 'model' -%}
-      <product-model class="deferred-media media media--transparent no-js-hidden" style="--ratio-percent: 100%;" data-media-id="{{ media.id }}">
+      <product-model class="deferred-media media media--transparent no-js-hidden" data-media-id="{{ media.id }}">
```

Comment:
By default we preserve the existing 1:1 ratios for models, but you are correct, when using media fit: fill there are scenarios we they can diverge from that. I believe these cases a fairly minimal as I understand, but I'll outline them below for clarity. Let me know if I'm missing any.

1. In cases of `media_fit: fill` and `desktop_layout: 2 columns` and model shares a row with a _portrait_ image. If the other image on the same row is a landscape or also square, the model media shouldn't crop. Particularly if 1:1 is the most common product image ratio, this should be an infrequent combination.
<img src="https://screenshot.click/08-22-zn5j1-xs2y3.png" width="200" />

2. In cases of `media_fit: fill` and `constrain: true` and viewport is small enough to constrain even square images. This can happen on tablet/desktop whenever models render fullwidth, where they can feasibly be large enough to be constrained as 1:1 images. I don't think this feel this will be _super_ common either, but I do I think this is more likely to occur than the other scenario, especially if we were to default the constrain setting to true/on.
<img src="https://screenshot.click/08-13-wrs4r-l2nan.png" width="200" />

> If we were to force 3D models to be square would add a layer of "magic" to the setting and make it less clear

Just want to mention this is an option of course, particularly for cases when the models are fullwidth (anything but desktop layout: 2 columns), but it would indeed be an unpredictable exception to the setting, which as I understand is our main issue with it.